presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -405,6 +406,54 @@ public WindowNode window(WindowNode.Specification specification, Map<Symbol, Win
                 0);
     }
 
+    public OutputNode output(Consumer<OutputBuilder> outputBuilderConsumer)
+    {
+        OutputBuilder outputBuilder = new OutputBuilder();
+        outputBuilderConsumer.accept(outputBuilder);
+        return outputBuilder.build();
+    }
+
+    public class OutputBuilder
+    {
+        private PlanNode source;
+        private List<String> columnNames = new ArrayList<>();
```

Comment:
Let's leave it. It is consistent with other builders and is converted to immutable collection anyway as soon as plan node is built